### DISCUSSION TITLE
Is there any way how to display all languages from one column on blade ?

### DISCUSSION THREAD

**Discussion by matej-1234** [spatie/laravel-translatable#326] [Q&A] [answered]
Hello everyone, currently I want to display all languages from one column, here you can see what I have stored in database :
![image](https://user-images.githubusercontent.com/71707914/154356060-60372f62-5821-4f89-9a7d-09dc0784ffa6.png)

and i want to display both sk and en in one row under each one. Here is an example how its done currently, but i am still loosing one of those languages in the blade:

![image](https://user-images.githubusercontent.com/71707914/154356207-f6cdab4d-56a1-48b0-ba13-461d1ee993ef.png)

Here is current blade:
![image](https://user-images.githubusercontent.com/71707914/154356392-debf97e8-d3f7-4fbc-9cab-63c7545c3b89.png)

Is there any function to give me all languages to print on blade ? 

Thanks in advance

**Comment by matej-1234** [comment]
Ah nice , i figured it out by my own, here is the solution:

![image](https://user-images.githubusercontent.com/71707914/154356988-9b36e525-78a8-49fc-8b04-6a279c5bcbaf.png)
and here is the output on the website:
![image](https://user-images.githubusercontent.com/71707914/154357228-31bb9467-5f55-4433-bb05-3576d909b550.png)

**Comment by anibalealvarezs** [comment, marked as answer]
Hello.

You can also create a mutator like this one (note the "s" at the end of "titles").
The ```getRawOriginal``` method will let you get the column value without the casting made by the ```hasTranslations``` trait.

```php
public function getTitlesAttribute($value)
{
    return json_decode($this->getRawOriginal('title'), true);
}
```

So, your ```$thesi->translations->titles``` will be an array of the type
```php
[
    "sk" => "sk text",
    "en" => "en text"
]
```

So, in your blade file you can print something like
```php
@foreach($thesi->translations->titles as $title)
    <span>{{ $title }}</span><br>
@endforeach
```

And your output HTML would be
```html
<span>sk text</span><br>
<span>en text</span><br>
```

Maiking it dynamic you will be able to add new languages without having to edit your code at all.

**Reply by matej-1234** [reply]
Thank you very much
